src/website/src/app/documentation/demos/datagrid/utils/pokemon-comparator.ts

```diff
@@ -8,6 +8,14 @@ import { User } from '../inventory/user';
 
 export class PokemonComparator implements ClrDatagridComparatorInterface<User> {
   compare(a: User, b: User) {
-    return a.pokemon.number - b.pokemon.number;
+    const aPokemon = a.pokemon.name.toLowerCase();
```

Comment:
Please take a look at the expandable datagrid demo. Sorting is just working fine. The issue was that we simply forgot to indicate that we are using the number property of Pokemon object to sort them like in the custom sorting demo: https://clarity.design/documentation/datagrid/custom-sorting.

So introducing this change will break other demos that sort items using the number property of Pokemon object